roles/ee_builder/tasks/00_build_ee.yml

```diff
@@ -42,7 +42,7 @@
 - name: Run the Ansible Builder Program
   ansible.builtin.command: >
     ansible-builder build -f
-      {{ ee_builder_dir | default(build_dir.path) }}/execution_environment.yml
+      execution_environment.yml
```

Comment:
I wonder while we are at it if we should update it so it uses execution-environment.yml vs execution_environment.yml which then would auto pickup by builder

Reply:
@sean-m-sullivan @Tompage1994

Reply:
looks good, had to double check, but for this task, it is indeed redundant, and we could change the name,